bundles/org.openhab.binding.mqtt.espmilighthub/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<update:update-descriptions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:update="https://openhab.org/schemas/update-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/update-description/v1.0.0 https://openhab.org/schemas/update-description-1.0.0.xsd">
+
+	<thing-type uid="mqtt:rgb_cct">
+		<instruction-set targetVersion="1">
+			<remove-channel id="level"/>
+			<update-channel id="colorTemperature">
+				<type>system.color-temperature</type>
```

Comment:
```suggestion
				<type>system:color-temperature</type>
```

Please check others. It might be worth verifying the file against the XSD after fixing these. I was checking because I was unsure if `update-channel`/`add-channel` can be freely mixed or had to be in specific order (add/update/remove) and found these issues.

I'm wondering if the upgrade actually worked despite these issues?

Reply:
I corrected the XML, the type references, ran the XML through a schema validator, and also recreated my things with the current main, then added with this PR, and confirmed my channels properly changed type on existing things.